src/test/java/org/jitsi/rtp/rtp/RtpPacketTest.kt

```diff
@@ -32,7 +38,7 @@ class RtpPacketTest : ShouldSpec() {
         // Timestamp: 1710483662
         0x65, 0xf3, 0xe8, 0xce,
         // SSRC: 839852602
-        0x32, 0x0f, 0x22, 0x3a,
+        0x48, 0x0f, 0x22, 0x3a,
```

Comment:
mind updating the comment which notes the ssrc encoded there? (since it looks like it's changing) (and below)